### PR TITLE
docs(manual): Update CLI User Manual

### PR DESCRIPTION
**Pull request by MuyangAmigo**

### CODE REVIEW

**Review by chagong — APPROVED** [review]

File: docs/cli/user-manual.md [hunk 1]
```diff
@@ -1,293 +1,284 @@
...
+| Parameters  | Required | Descriptions |
+|:----------------  |:-------------|:-------------|
+|`--app-name` | Yes| Name of your Teams application.|
+|`--interactive`| No | Select the options interactively. Options are `true` and `false`. The default value is `true`.|
+|`--capabilities`| No| Choose Teams application capabilities, multiple Options are: `tab`, `bot`, `messaging-extension` and `tab-spfx`. The default value is: `tab`.|
+|`--programming-language`| No| Programming Language for the project. Options are `javascrip` or `typescript` and default value is: `javascript`.|
+|`--folder`| No | Project directory. A sub folder with the your app name will be created under this directory. The default value is: `./`.|
+|`--spfx-framework-type`| No| Applicable if `Tab(SPfx)` capability is selected. Frontend Framework. Options are `none` and `react`, the default value is: `none`.|
+|`--spfx-web part-name`| No | Applicable if `Tab(SPfx)` capability is selected. Web part Name. The default value is: "helloworld". |
+|`--spfx-web part-desp`| No | Applicable if `Tab(SPfx)` capability is selected. Web part Description. The default value is: "helloworld description". |
```
Comment (Alive-Fish) [inline docs/cli/user-manual.md:55]: There is one more space here.

File: docs/cli/user-manual.md [hunk 2]
```diff
@@ -1,293 +1,284 @@
...
-Manage cloud service accounts. The supported cloud services are `azure` and `m365`.
+Manage cloud service accounts. The supported cloud services are `Azure` and `Microsoft 365`.
 
-## Commands
 | `teamsFx account` Commands  | Descriptions |
 |:----------------  |:-------------|
 | `teamsfx account login <service>`      | Log in to the selected cloud service. |
 | `teamsfx account logout <service>`      | log out of selected cloud service. |
-| `teamsfx account set`      | Update account settings. |
+| `teamsfx account set --subscription`      | Update account settings to set a subscription ID. |
```
Comment (Alive-Fish) [inline docs/cli/user-manual.md:89]: There may miss the `teamsfx account show` command.

File: docs/cli/user-manual.md [hunk 3]
```diff
@@ -1,293 +1,284 @@
...
+| Parameters  | Required | Descriptions |
+|:----------------  |:-------------|:-------------|
+|`--local`| No | Preview the application from local. `--local` is exclusive with `--remote`. |
+|`--remote`| No | Preview the application from remote. `--remote` is exclusive with `--local`. |
+|`--env`| No | Select an existing environment for the project when parameter `--remote` appended. |
+|`--folder`| No | Project root directory. The default value is `./`. |
+|`--browser`| No | The browser to open Teams web client. Options are `chrome`, `edge` and `default` (system default browser). The default value is `default`. |
+|`--browser-arg`| No | Argument to pass to the browser, requires --browser, can be used multiple times (e.g. --browser-args="--guest") |
+|`--sharepoint-site`| No | SharePoint site URL, like `{your-tenant-name}.sharepoint.com` (only for SPFx project remote preview). |
+
```
Comment (Alive-Fish) [inline docs/cli/user-manual.md:226]: Miss `env` here.

File: docs/cli/user-manual.md (hunk 2 above)
Comment (MuyangAmigo) [reply docs/cli/user-manual.md:90]: Thanks, added!

File: docs/cli/user-manual.md (hunk 1 above)
Comment (MuyangAmigo) [reply docs/cli/user-manual.md]: Updated.

**Review by Alive-Fish — APPROVED** [review]

**Pull request merged**